### PR TITLE
fix address book init lifecycle to work with permissions check

### PR DESCRIPTION
**Pull request by ele828**

### CODE REVIEW

**Comment by marcusrc** [comment]
Something went wrong. Investigate!

File: src/modules/AddressBook/index.js [hunk 1]
```diff
@@ -163,7 +163,13 @@ export default class AddressBook extends Pollable {
       if (this._shouleCleanCache()) {
         this._cleanUp();
       }
-      await this._initAddressBook();
+      if (this._hasPermission) {
```
Comment (embbnux) [inline src/modules/AddressBook/index.js]: Please await permission to be ready

**Comment by marcusrc** [comment]
Something went wrong. Investigate!

File: src/modules/AddressBook/index.js (hunk 1 above)
Comment (ele828) [reply src/modules/AddressBook/index.js]: fixed

**Comment by marcusrc** [comment]
Something went wrong. Investigate!

**Pull request merged**